tests/ctst/common/hooks.ts

```diff
@@ -17,7 +17,7 @@ import { cleanS3Bucket } from './common';
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
 
 const { atMostOnePicklePerTag } = parallelCanAssignHelpers;
-const noParallelRun = atMostOnePicklePerTag(['@AfterAll', '@PRA', '@ColdStorage']);
+const noParallelRun = atMostOnePicklePerTag(['@AfterAll', '@PRA', '@ColdStorage', '@Utilization']);
```

Comment:
As per guidelines, you should not be using `atMostOnePicklePerTag` otherwise it means that you test breaks rule #1 of idempotency..

Reply:
Yes, not needed since we lock the execution of the jobs, I removed it and tagged all the scenarios appropriately